### ISSUE TITLE
Cannot Handle Popups

### ISSUE THREAD

**Issue by hwasiti** [scoute-dich/browser#58]
A lot of websites depends on popups in login especially those using OAth Authentication.

In this browser, popup will be opened on the same Webview and destroy the webview.opener (the original webpage).

Suggestion: to override opening new window and let another tab handle it. Assign the original window as the webview.opener in the popup window so it can return back the JSON data that have been received in the popup from the success of authentication of the user.

**Comment by hwasiti** [comment]
For example try to login:
[http://www.feedly.com](http://www.feedly.com)

**Comment by scoute-dich** [comment]
At the moment I'm working on some big changes. I'm trying to use Ninja Browser as base and then implementing all of the functionality from my own work. It seems that the feedly login will work in the next version. Of you want to test it, I can upload an apk file.

**Comment by gigabytedevelopersinc** [comment]
Cool.. I've been seeing you speak of this. May I have a look at the apk?

On Dec 2, 2017 2:01 PM, "Gaukler Faun" <notifications@github.com> wrote:

At the moment I'm working on some big changes. I'm trying to use Ninja
Browser as base and then implementing all of the functionality from my own
work. It seems that the feedly login will work in the next version. Of you
want to test it, I can upload an apk file.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/scoute-dich/browser/issues/58#issuecomment-348690723>,
or mute the thread

**Comment by hwasiti** [comment]
Yes I am interested to test your apk.

I have tested Ninja browser and Cornowser and ultimate browser. All of them do not support popups. Surprisingly, so far I have not found any open source browser supporting popup only this one:
https://github.com/anthonycr/Lightning-Browser

Lightning browser supports popups. You can test it to login in feedly for example.

**Comment by scoute-dich** [comment]
[app-debug.zip](https://github.com/scoute-dich/browser/files/1542074/app-debug.zip)
**Warning: nearly all of your app data will be lost.** 
Here's the test apk. Just replace zip with apk. Some things must be done before releasing:
- strings (English and German are messed up)
- new Help dialog
- what you tell me

Thanks for testing.

**Comment by scoute-dich** [comment]
Update is on his way. Since I switched to Ninja as new base of Browser please test the new implementations and open a new issue if there are further problems. Thanks for your patience.
